src/main/resources/sqlscripts/tbls_PersistentQ_multiplecounter_v1.0.sql

```diff
@@ -2,21 +2,25 @@
 
 -- =============================================
 -- Author:		Mohammed Hafejee
---				
+--
 -- Create date: 10-JUN-2020
--- Description:	Script to create base table used by ECM multiples persistent queue 
+-- Description:	Script to create base table used by ECM multiples persistent queue
 -- Called by  : fn_persistentQ_getNextMultipleCountVal
 -- VERSION	:	10-JUN-2020		1.0  - Initial
 -- =============================================
 
-/***********   multipleCounter   ************/  
+/***********   multipleCounter   ************/
 
 DROP TABLE IF EXISTS multipleCounter;
-CREATE TABLE multipleCounter 
+CREATE TABLE multipleCounter
   (
   multipleRef  varchar(25),
   counter integer DEFAULT 1
   );
 
+ALTER TABLE multipleCounter
```

Comment:
why is counter part of the primary key? this would then permit the following rows
12345, 0
12345, 1
12345, 2
etc

Reply:
good point, fixed.